semeio/jobs/scripts/design2params.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+#
+import argparse
+import logging
+import os
+import sys
+
+from semeio import valid_file
+from semeio.jobs.design2params import design2params
+
+description = """
+Reads a design matrix in XLSX-format and:
+ * Converts specified worksheet to txt-format and puts designmatrix.txt in RUNPATH
+ * Fetches values belonging to given ERT realization and *appends*
+   key-value pairs to parameters.txt
+ * Creates designparameters.txt which only contains variables taken from the design matrix, this
+   can be given to CUSTOM_KW
+Requires a matrix with column header as strings in topmost row in Excel.
+Row 2 in Excel must then correspond with the data you want for 'realization 0'
+Column 1 in Excel should contain the realization number, and will be ignored by this script.
+You must run this script as a FORWARD_MODEL from your ERT config, before
+you run DESIGN_KW.
+
+Usage:
+ $ design2params.py <realization-id> <xls-filename> <worksheet-name> [<defaultssheetname>]
+defaultssheetname is an optional argument, you may choose not to provide it (just give an empty string)
```

Comment:
`you may choose not to provide it (just give an empty string)` being optional you shouldn't have to provide an empty string?

Reply:
An on the command line at least you don't have to give an empty string either.